src/app.ts

```diff
@@ -34,10 +36,13 @@ app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerSpec))
  *       200:
  *         description: Réponse réussie
  */
+
 app.get('/', (_: Request, res: Response) => {
-  res.send('Hello, Express with TypeScript!')
+  res.send('Hello, Express with TypeScript!!!! ')
 })
 
+app.use('/users', usersRouter)
```

Comment:
```suggestion
app.use('/user', usersRouter)
```

since related routes are only put and delete and there are only applying to current user, it should not be plural.

also, apply the authenticate middleware on it. so that only the logged in user can access this route